middlewares/confirm-mail-changePassword.js

```diff
@@ -0,0 +1,52 @@
+const nodemailer = require("nodemailer");
+const User = require("../models/User");
+
+const sendPasswordChangeConfirmationEmail = async (email) => {
+  console.log(email);
+  try {
+    const transporter = nodemailer.createTransport({
+      service: "Gmail",
+      auth: {
+        user: "pruebasdanielmayo@gmail.com",
+        pass: "tfehpymjjkfwsagn",
```

Comment:
I'm not sure, but I think here, you should pass the `user.email` not your email, am I right?

Same thing in `pass`